package.json

```diff
@@ -11,6 +11,7 @@
     "dependencies": {
         "bluebird": "^2.9.9",
         "lodash": "^3.2.0",
+        "node-tech-logger": "git://github.com/AirVantage/node-tech-logger.git#1.0.0",
```

Comment:
If I want to use v2.x, I need to update all the client projects of node-tech-http, and all their clients, and all their clients until I reach av-server and av-accounts (and av-metrics, and so on...) to use logger 2.x.
Maybe. But not now.